### PR TITLE
Installing SQL server PS module

### PR DESCRIPTION
**Pull request by Ajay-MS**
SQLPS no longer being updated. The most up-to-date PowerShell module is the SqlServer module.

**Reference** 
https://docs.microsoft.com/en-us/sql/powershell/download-sql-server-ps-module?view=sql-server-2017

### CODE REVIEW

**Comment by Ajay-MS** [comment]
Moved validation logic to separate file

File: images/win/scripts/Installers/Validate-SQLPowershellTools.ps1 [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    $sqlPSVersion = $modules.Version
+}
+
+# Validate the SQLserver PS module installation
+$modules = Get-Module -Name SQLServer -ListAvailable
+Write-Host "The SQLServer Modules present are:"
+$modules | Select-Object Name,Version,Path | Format-Table
+
+if ($modules) {
+    $sqlServerPSModuleVersion = $modules.Versionss
```
Comment (lokesh755) [inline images/win/scripts/Installers/Validate-SQLPowershellTools.ps1]: I think it should be "$modules.Version"

File: images/win/vs2017-Server2016-Azure.json [hunk 2]
```diff
@@ -284,6 +284,12 @@
                 "{{ template_dir }}/scripts/Installers/Install-Svn.ps1"
             ]
         },
+        {
+            "type": "powershell",
+            "scripts":[
+                "{{ template_dir }}/scripts/Installers/Validate-SQLPowerShellTools.ps1"
```
Comment (lokesh755) [inline images/win/vs2017-Server2016-Azure.json]: Validate script should be placed after corresponding Install script. Also there should be minimum one restart between Install and Validate scripts.

**Comment by Ajay-MS** [comment]
@lokesh755 Thanks for your review comments. I have resolved both.

**Pull request merged**